std_msgs/UInt8 home
std_msgs/UInt8 land
std_msgs/UInt8 hover
std_msgs/UInt8 takeOff
